feature/troubleshooting/src/main/java/com/simprints/feature/troubleshooting/overview/OverviewViewModel.kt

```diff
@@ -22,6 +26,7 @@ internal class OverviewViewModel @Inject constructor(
     private val collectLicenseStates: CollectLicenceStatesUseCase,
     private val collectNetworkInformation: CollectNetworkInformationUseCase,
     private val doServerPing: PingServerUseCase,
+    private val doExportLogs: ExportLogsUseCase,
```

Comment:
`exportLogs`?

Reply:
It separates the "action" use cases from the data collection ones.